### PR TITLE
[batch] Use premium SSD data disks in Azure

### PR DESCRIPTION
**Pull request by jigold**

### CODE REVIEW

**Comment by jigold** [comment]
Bump. This needs to go into production before we do massive scale tests since Standard_SSD charges per IO operation.

**Comment by jigold** [comment]
Actually, this seems to cause instances to hang when using an actual data disk. I'm going to put the WIP tag on until I get a chance to investigate

**Review by danking — CHANGES_REQUESTED** [review]
dismiss when you're ready for a new look

**Comment by jigold** [comment]
I'm not sure if we need a Premium SSD on the os disk or not.

**Comment by jigold** [comment]
See question, but otherwise tests are passing.

File: batch/batch/cloud/azure/resource_utils.py [hunk 1]
```diff
@@ -90,7 +89,7 @@ def azure_worker_properties_to_machine_type(worker_type: str, cores: int, local_
         return f'Standard_F{cores}s_v2'
 
     if local_ssd_data_disk:
-        additive_features = 'd'
+        additive_features = 'ds'
```
Comment (jigold) [inline batch/batch/cloud/azure/resource_utils.py:92]: d means use a local SSD data disk. s means support premium disks.

**Review by danking — DISMISSED** [review]
x

**Review by danking — APPROVED** [review]

**Pull request merged**